### PR TITLE
Fix artifact upload error message and HTTP status code on error.

### PR DESCRIPTION
**Pull request by kjaskiewiczz**

### CODE REVIEW

File: resources/images/model/images.go [hunk 1]
```diff
@@ -114,10 +114,8 @@ func (i *ImagesModel) handleArtifact(
 	metaArtifactConstructor, err := getMetaFromArchive(&tee)
 	if err != nil {
 		pW.Close()
-		if uploadResponseErr := <-ch; uploadResponseErr != nil {
-			return "", controller.ErrModelArtifactUploadFailed
-		}
-		return "", controller.ErrModelInvalidMetadata
+		_ = <-ch
```
Comment (bboozzoo) [inline resources/images/model/images.go]: IMO this line could just read:
```
<-ch
```

Comment (kjaskiewiczz) [inline resources/images/model/images.go]: done

**Comment by bboozzoo** [comment]
rebase?

**Comment by maciejmrowiec** [comment]
1.0.2 candidate ? @kjaskiewiczz

**Comment by bboozzoo** [comment]
closing this for now, I'll be opening a PR which incorporates this patch

**Pull request closed without merging**